editor/editor_properties_array_dict.cpp

```diff
@@ -593,7 +593,7 @@ void EditorPropertyDictionary::_property_changed(const String &p_prop, Variant p
 		Variant key = dict.get_key_at_index(idx);
 		dict[key] = p_value;
 
-		emit_signal("property_changed", get_edited_property(), dict);
+		emit_signal("property_changed", get_edited_property(), dict, true);
```

Comment:
I tried setting this to `changing` and the cursor stucks while editing Vector2 by dragging.